plugin-server/src/worker/vm/extensions/geoip.ts

```diff
@@ -1,12 +1,15 @@
 import { GeoIPExtension } from '@posthog/plugin-scaffold'
 
 import { Hub } from '../../../types'
-import { fetchIpLocationInternally } from '../../mmdb'
 
 export function createGeoIp(server: Hub): GeoIPExtension {
     return {
         locate: async function (ipAddress) {
```

Comment:
keeping the same signature for compatibility, although this function does not need to be async anymore